Extension/src/LanguageServer/extension.ts

```diff
@@ -106,7 +109,7 @@ async function lookupIncludeInVcpkg(document: vscode.TextDocument, line: number)
     if (!matches || !matches.length || !matches.groups) {
         return [];
     }
-    const missingHeader: string = matches.groups.includeFile.replace(/\//g, '\\');
+    const missingHeader: string = matches.groups.includeFile.replace(/[\\\/]/g, '\\');
```

Comment:
I don't think this line needs to change. Do you know what was the problem here?  You just added backslash to the list of things to replace, but you'd just replace it with a backslash so that doesn't change the string.

Reply:
I see I thought the regex was slightly wrong, but I just misinterpreted how JavaScript applies backslashes.